.github/dependabot.yml

```diff
@@ -0,0 +1,64 @@
+version: 2
```

Comment:
doesn't this file needs to be in the `workflow` folder?

Reply:
I don't believe so. The docs only mention adding it to `.github` (https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#:~:text=You%20must%20store%20this%20file%20in%20the%20.github%20directory%20of%20your%20repository.)

From what I have read, dependabot is added to the workflow folder on cases where we want verision updates for the actions themselves (https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#enabling-dependabot-version-updates-for-actions)